src/main/java/dekk/pw/pokemate/tasks/CatchPokemon.java

```diff
@@ -55,7 +55,7 @@ public void run() {
                 if (target != null && pokeball != null) {
                     Walking.setLocation(context);
                     EncounterResult encounterResult = target.encounterPokemon();
-                    if (encounterResult.wasSuccessful()) {
+                    if (encounterResult.wasSuccessful() && !Config.getIgnoreCatchingPokemon().contains(target.getPokemonId().getNumber())) {
```

Comment:
Thanks for the feedback,Added the filter in the catchable pokemon list instead so if we don't have any pokemons to catch it will also wont try to pick a pokeball.